urlwatch

```diff
@@ -168,8 +172,10 @@ if __name__ == '__main__':
     parser.add_option('-q', '--quiet', action='store_true', dest='quiet', help='Avoid diff output on stdout (--mailto)')
     parser.add_option('-A', '--auth', dest='auth', action='store_true', default=False, help='Enable authentication from keyring for SMTP')
     parser.add_option('-c', '--cache', dest='cache', metavar='DIR', help='Use specified DIR as cache directory.', default=None)
+    parser.add_option('-P', '--pushover', dest='pushover_file', metavar='FILE', help='Read Pushover login details from [FILE] instead')
```

Comment:
"...instead"? I think the help text should be something like "Use Pushover service with login details from FILE".